scripts/visualization/opf_visualizer.js

```diff
@@ -331,6 +331,20 @@ angular.module('app').controller('AppCtrl', ['$scope', '$timeout', function($sco
     }
   };
 
+  // this function highlights a range in the graph
+  var highlightPoint = function(canvas, area, g) {
+              var bottom_left = g.toDomCoords(20, -20);
+              var top_right = g.toDomCoords(40, +20);
```

Comment:
@breznak I think this is because g.toDomCoords takes as arguments references to the data, while you are using hard-coded values. So, depending on what series the data point is on, it might return strange things using hard-coded values like that.